config/application.yml

```diff
@@ -0,0 +1,22 @@
+# Add configuration values here, as shown below.
+#
+# pusher_app_id: "2954"
+# pusher_key: 7381a978f7dd7f9a1117
+# pusher_secret: abdc3b896a0ffb85d373
+# stripe_api_key: sk_test_2J0l093xOyW72XUYJHE4Dv2r
+# stripe_publishable_key: pk_test_ro9jV5SNwGb1yYlQfzG17LHK
+#
+# production:
+#   stripe_api_key: sk_live_EeHnL644i6zo4Iyq4v1KdV9H
+#   stripe_publishable_key: pk_live_9lcthxpSIHbGwmdO941O1XVU
+
+# Available providers: developer, saml, azure_activedirectory_v2
```

Comment:
I'd like these configuration options to not apply to all environments, as this would make these available on our production deploy without overriding the file.

Maybe you could comment out the variables and use something like:
```yaml
test:
  OMNIAUTH_PROVIDERS: "developer,saml,azure_activedirectory_v2"
development:
  OMNIAUTH_PROVIDERS: "developer"
production:
  OMNIAUTH_PROVIDERS: ""
```

Reply:
Added.